plugins/bulk_upload/drop_folder_xml/plugins/batch/Engine/KFeedDropFolderEngine.php

```diff
@@ -275,4 +275,44 @@ protected function getMaxFeedBitrate (SimpleXMLElement $feedItem)
 		
 		return max ($bitrates);
 	}
+	
+	/**
+	 * @param string $url
+	 * @return string
+	 */
+	protected function fetchFeedContent ($url)
+	{
+		$user = parse_url ($url, PHP_URL_USER);
+		if (!is_null ($user))
+		{
+			$password = parse_url ($url, PHP_URL_PASS);
+			$user = parse_url($url, PHP_URL_USER);
+			$password = parse_url($url, PHP_URL_PASS);
+			
+			$protocol = parse_url($url, PHP_URL_SCHEME);
+			$hostname = parse_url($url, PHP_URL_HOST);
+			$port = parse_url($url, PHP_URL_PORT);
+			$params = parse_url($url, PHP_URL_PATH);
+			$queryArgs = parse_url($url, PHP_URL_QUERY);
+
+			$url =  "$protocol://$hostname" .  ($port? ":$port" : "") . "$params?$queryArgs";
+		}
+		
+		$ch = curl_init ($url);
+		if (!is_null ($user))
```

Comment:
@hilak  - why not add those lines inside the if statement in line 286? it is the same condition...